src/main/java/org/sonarlint/intellij/config/global/SonarQubeServerMgmtPanel.java

```diff
@@ -325,8 +325,9 @@ private void setStatus(ConnectedSonarLintEngine.State state) {
         break;
       case UPDATING:
         builder.append("updating..");
-
         break;
+      case NEED_UPDATE:
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) End this switch case with an unconditional break, return or throw statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS128)